pkg/qrcodegenerator/qrcode/enddevice/lora_alliance_tr005.go

```diff
@@ -143,14 +151,52 @@ func (m *LoRaAllianceTR005) UnmarshalText(text []byte) error {
 	return m.Validate()
 }
 
-// AuthenticatedEndDeviceIdentifiers implements the AuthenticatedEndDeviceIdentifiers interface.
-func (m *LoRaAllianceTR005) AuthenticatedEndDeviceIdentifiers() (joinEUI, devEUI types.EUI64, authenticationCode string) {
-	return m.JoinEUI, m.DevEUI, m.OwnerToken
+// GetEntityOnboardingData implements the Data interface.
+func (m *LoRaAllianceTR005) GetEntityOnboardingData() *ttnpb.EntityOnboardingData {
+	paths := []string{
+		"ids",
+		"claim_authentication_code",
+	}
+	attributes := make(map[string]string)
```

Comment:
> If the QR code contains a non-zero Vendor ID and Profile ID, lookup the LoRaWAN device profile (https://github.com/TheThingsNetwork/lorawan-stack/issues/4842) and put this in the onboarding state

I'm using attributes here since these fields are not defined in the `EndDevice` message. The next step of the onboarding flow for the client need these int values  so I'm using hex encoded string. The caller can then do a `strings.Atoi` and use that to fetch the device profile by numeric ID.